src/query/config/src/inner.rs

```diff
@@ -167,6 +163,18 @@ pub struct QueryConfig {
     pub table_cache_bloom_index_meta_count: u64,
     /// Max number of cached bloom index filters
     pub table_cache_bloom_index_filter_count: u64,
+    /// Max size of in memory table column object cache
+    pub table_cache_column_mb_size: u64,
```

Comment:
Suggest this config name to `table_cache_column_max_size`(or another better name), in bytes?  `table_disk_cache_mb_size` same.

Some day our config value may be: `1MB|GB` , it's hard to map them in the underlying.